Dockerfile

```
@@ -0,0 +1,37 @@
+# Build stage
+FROM ghcr.io/userver-framework/ubuntu-22.04-userver-pg-dev:latest AS builder
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)